web_editor/js/main.js

```diff
@@ -112,44 +85,47 @@ export class LaneEditor {
 
     document.getElementById("diff-tags-table").innerHTML = output;
   }
-}
-
-function setupOnce() {
-  // Setup the toolbox controls
-  new Sortable(document.getElementById("toolbox"), {
-    group: {
-      name: "toolbox",
-      pull: "clone",
-    },
-    sort: false,
-    animation: 150,
-    ghostClass: "card-being-dragged",
-  });
 
-  // TODO Disable the ghostClass of lanes when going here
-  new Sortable(document.getElementById("delete"), {
-    group: "lanes",
-    onAdd: function (evt) {
-      var el = evt.item;
-      el.parentNode.removeChild(el);
-    },
-  });
-
-  document.getElementById("start-editing").onclick = async function () {
-    try {
-      window.app = await LaneEditor.create();
-      window.app.render();
-    } catch (err) {
-      window.alert(`Error: ${err}`);
-    }
-  };
-  document.getElementById("diff-tags").onclick = function () {
-    if (window.app) {
+  #setupButtons() {
+    document.getElementById("diff-tags").onclick = () => {
       try {
-        window.app.diffTags();
+        this.#diffTags();
       } catch (err) {
         window.alert(`Error: ${err}`);
       }
-    }
-  };
+    };
+
+    document.getElementById("new-driving").onclick = () => {
+      this.road.lanes.push({
```

Comment:
Always appending to the end is clunky. You had suggested having + buttons in between each pair of lanes, and presumbly a way to click the type and change it. I like that idea now that I type it out; I'll try that in a followup